index.html

```diff
@@ -46,32 +47,34 @@
             </li>
           </ul>
         </div>
+        <!-- did mistake ... fix -->
+        <button class="donate_now">Donate Now</button>
+      
+      </header>
 
-        <button>Donate Now</button>
-
-    </div>
 
     <div class="content">
-        <div class="main">
+        <main class="main">
           <div class="alert">
             You've helped us give <span class="text-highlight">72</span> bikes
             to refugees and asylum seekers so far. <strong>Thank you!</strong>
           </div>
 
           <div class="hero">
+            <!-- still looking how to add background bike photo , nevermind founded :)-->
             <h1>Bikes for Refugees</h1>
             <p>
               Providing donated bikes and accessories to refugees and asylum
-              seekers in Scotland.
+              seekers in England.
```

Comment:
Good catch :D I don't think it was intended though. Being detailed is a good asset in programming.